en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-example.md

```diff
@@ -14,11 +14,11 @@ If you do not want to configure this yourself, you can simply [get the project](
 
 ## Setting up the environment 
 
-1. Please follow the steps mentioned in the [Setting up the Amazon S3 Environment ]({{base_path}}/reference/connectors/amazonsqs-connector/amazonsqs-connector-config) document in order to create a Amazon account and obtain access key id and secret access key. Keep them saved to be used in the next steps.  
+1. Please follow the steps mentioned in the [Setting up the Amazon S3 Environment ]({{base_path}}/reference/connectors/amazonsqs-connector/amazonsqs-connector-config) document in order to create an Amazon account and obtain access key id and secret access key. Keep them saved to be used in the next steps.  
 
 2. In this example we will be using XPath 2.0 which needs to be enabled in the product as shown below before starting the integration service. 
 
-    If you are using the Micro Integrator of **EI 7** or **APIM 4.0.0**, you need to enable this property by adding the following to the PRODUCT-HOME/conf/deployment.toml file. You can further refer to the [Product Configurations](https://apim.docs.wso2.com/en/4.2.0/reference/config-catalog/#http-transport).
+    If you are using the Micro Integrator of **EI 7** or **MI 4.x.0**, you need to enable this property by adding the following to the PRODUCT-HOME/conf/deployment.toml file. You can further refer to the [Product Configurations](https://apim.docs.wso2.com/en/4.2.0/reference/config-catalog/#http-transport).
```

Comment:
It should be MI 4.0.0. This is referring to older versions.

Reply:
Let's point to the config catalog content in this doc space

Reply:
Fixed with dc82fd313f75013cb3fc9b1578a14b94ad564545